src/Files.App (Package)/Package.appxmanifest

```diff
@@ -119,12 +119,26 @@
                 </uap3:Extension>
 
                 <uap:Extension Category="windows.protocol">
+                    <!--  This is kept for backward compatibility.  -->
                     <uap:Protocol ReturnResults="none" Name="files-uwp" />
                 </uap:Extension>
 
+                <uap:Extension Category="windows.protocol">
+                    <!--  This is kept for backward compatibility.  -->
+                    <uap:Protocol ReturnResults="none" Name="files" />
```

Comment:
Can this be moved to line 124?

Reply:
According to the docs, it indicates you can't put more than one child element to uap:Extension element.